app/controllers/listing_images_controller.rb

```diff
@@ -23,6 +23,7 @@ def add_from_url
     url = escape_s3_url(params[:path], params[:filename])
 
     if !url.present?
+      logger.warn("No image URL provided", :no_image_url_provided, params)
```

Comment:
I feel this should be either info level (Everything is ok, client messed up but we noticed and fixed it) or error (An error happened here but was caught and handled). Warn would then be something like "this field really should have a value but it doesn't, I'm gonna save it anyway but later on you will see broken images as result"). So Warn is a warning that something is not right and that can potentially lead to problems later.